package.json

```diff
@@ -19,7 +19,7 @@
     "storybook": "nuxt storybook",
     "test": "jest",
     "test:watch": "jest --watch",
-    "test:unit": "jest",
+    "test:unit": "jest --collectCoverage=false",
```

Comment:
Why not collect coverage?

Reply:
While writing the tests, I don't wanted to see the coverage table every time (it's long!), only the test results. The `test` and `test:unit` command are the same so I though we can modify one to skip the table. I can revert it if this was made for a specific purpose of course.